src/legend_data_monitor/subsystem.py

```diff
@@ -547,8 +539,18 @@ def get_parameters_for_dataloader(self, parameters: typing.Union[str, list_of_st
                 # otherwise just add the parameter directly
                 params.append(param)
 
+        # --- check if parameters have '~', if so remove for loading the corresponding lh5 parameter
```

Comment:
```~``` is needed for applying anti-QCs. BUT lh5 available parameters do not show that in the name's variable, so to load columns we need to remove that. The ```~``` is needed for the cuts we later apply in ```cuts.py```